meetings/2025-05-13-taskforce-meeting.md

```diff
@@ -0,0 +1,87 @@
+- Emelia: do we need an actual text property for content warning, or do we want the summary to BE the content warning?
+- Jaz: why can't I personally write a `summary` of "Trigger warning: content includes blah" -- I use Trigger warning specifically because that's different from CW.
+- Emelia: if we say `summary` is that, and then labels actually drive content warnings, how do you have software indicate "the author meant this as a CW" vs "the author meant to collapse".
+- a: for collapsing, there's the `sensitive` property. Mastodon doesn't distinguish between `summary` and `sensitive`
+- Jaz: let's take this out of mastodon and into a blog. What if a blog author writes warnings into the top of the post. That's fine. Let's have free text and have it not mean anything other than it's free text. We don't need a separate property for content warnings. 
+- Emelia: So perhaps expected behavior is: summary is free text; sensitive tells people whether things are collapsed or not; labels can be interpreted separately.
+- a: i still think that `summary` can be correct even for content warnings, it's more about how much do we want to be able to signal the purpose of it? in AS2 it is defined as a "summary" in the sense that a plaintext representation of an object would have it be the secondary bit next to the name/title 
+- Jaz: I have found `sensitive` to be nonsense. maybe we just have it all be labels
+- Darius: seems like `sensitive` is an example of the phase 2 "here is a recommended action"
+- Jaz: with structured labels we are going to want actions.
+- Jaz: can I get clarity. Emelia I think this is your contention, I am looking at FIRES labels. I think I understand that you won't or can't ascribe an Actor to the Activity of labeling a thing. I'm trying to wrap my head around how we do a spec where we talk about label discovery and representation but we don't add in who is responsible for applying labels.
+- Emelia: we could add `attributedTo`. In the case of FIRES I don't need it because the label _is_ the provider of the label but for a more generic case we could.
+- a: https://www.w3.org/TR/annotation-model/#motivation-and-purpose a Label could also extend from an Annotation, it can be multiple things depending on the properties you want it to have 
```

Comment:
```suggestion
- Emelia: We could add `attributedTo`. In the case of FIRES, I don't need it because the label _is_ the provider of the labe,l but for a more generic case we could.
- a:  A Label could also extend from an Annotation. It can be multiple things [depending on the properties you want it to have](https://www.w3.org/TR/annotation-model/#motivation-and-purpose).
```